addons/html_builder/static/src/builder.js

```diff
@@ -90,7 +90,12 @@ export class Builder extends Component {
                         editorBus.trigger("DOM_UPDATED");
                     }
                 },
-                reloadEditor: this.props.reloadEditor,
+                reloadEditor: (param = {}) => {
+                    this.props.reloadEditor({
+                        ...param,
+                        initialTab: this.state.activeTab,
```

Comment:
{
initialTab: this.state.activeTab
...param,
}

To make initialTab configurable